util/names.py

```diff
@@ -6,7 +6,10 @@
 from uuid import uuid4
 
 REPOSITORY_NAME_REGEX = re.compile(r"^[a-z0-9][\.a-z0-9_-]{0,254}$")
-
+# Extended repostitory name regex: allows "/" in repo names
+REPOSITORY_NAME_EXTENDED_REGEX = re.compile(
+    "^(?=.{0,255}$)[a-z0-9][\.a-z0-9_-]*(?:\/[a-z0-9][\.a-z0-9_-]*)*$"
```

Comment:
@kleesc - should this be a raw string (r"...") as well like the REPOSITORY_NAME_REGEX?